tests/unit-core/test-api-set-and-clear-error-flag.c

```diff
@@ -130,9 +185,21 @@ main (void)
   error = jerry_create_error_from_value (error, true);
   jerry_release_value (error);
 
+  value = jerry_create_number (42);
+  error = jerry_create_abort_from_value (value, true);
+  error = jerry_create_abort_from_value (error, true);
+  jerry_release_value (error);
+
+
```

Comment:
One newline here.

Reply:
Didn't see that, thank you. I updated the PR.